### PR TITLE
Log endpoint type and viewer id params

### PR DESCRIPTION
**Pull request by stulees**
## Description
Read `env` and `viewerId` values from URL query params injected by Viewer as per [design](https://docs.google.com/document/d/1g_JLSabdIGidQuv5uuw7Ev_roXvGDI4BX7raB0R9BAE/edit#)

Include `viewer_id` field in all logs.

When presentation type is `sharedschedule`, include `endpoint_type` field (based on value of `env`). 

**Note**
BQ widget tables will require adding two new fields: `viewer_id` and `endpoint_type`.

## Motivation and Context
Provide required data for Usage Analysis

## How Has This Been Tested?
TODO: Requires testing via Widget. 

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a

### CODE REVIEW

**Comment by stulees** [comment]
@olegrise This PR will be rebased once #161 is merged. Also, manual testing will need to be done when we update Widgets. Please review in the meantime, thanks

File: src/js/logger.js [hunk 1]
```diff
@@ -28,17 +29,23 @@ RiseVision.Common.LoggerUtils = (function() {
...
-        }
+
+      var parent = utils.getQueryParameter("parent"); // legacy presentations
+      var type = utils.getQueryParameter("type"); // templates
+      var env = utils.getQueryParameter("env"); // endpoint
+      var viewerId = utils.getQueryParameter("viewerId");
+
+      var presentation_type = type ? type : (parent ? utils.getQueryStringParameter("type", parent) : "");
+      var endpoint_type = env ? env : (parent ? utils.getQueryStringParameter("env", parent) : "");
+      var viewer_id = viewerId ? viewerId : (parent ? utils.getQueryStringParameter("viewerId", parent) : "");
```
Comment (olegrise) [inline src/js/logger.js:40]: @stulees I'm not sure this is needed. Viewer is not going to have `viewerId` URL param. We can leave it as is for the same of consistency. It won't hurt.

**Review by olegrise — APPROVED** [review]
Looks good

File: src/js/logger.js (hunk 1 above)
Comment (stulees) [reply src/js/logger.js:40]: @olegrise We spoke, the `viewerId` will be needed as a URL param, so this code should remain intact. Cheers

File: src/js/logger.js [hunk 2]
```diff
@@ -28,17 +29,23 @@ RiseVision.Common.LoggerUtils = (function() {
...
+      var schedule_id = parent ? utils.getQueryStringParameter("id", parent) : "";
+
+      json.viewer_id = viewer_id;
+
+      if (presentation_type ==="sharedschedule") {
+        json.presentation_type = presentation_type;
+        json.schedule_id = schedule_id;
+        json.endpoint_type = endpoint_type;
```
Comment (olegrise) [inline src/js/logger.js:48]: @stulees I wonder if calling this field the `endpoint_type` is a good idea. The way the Endpoint Type is described in the Env Verifier requirements and Logging Architecture is a combination of `type` & `env` parameters. The `env` URL parameter is sort of a subtype of the Shared Schedule type. I would just call it `env` so we don't mix terminology. What do you think?

**Comment by stulees** [comment]
@olegrise I made the change we discussed, please review last commit, thanks

**Review by olegrise — APPROVED** [review]
Looks good

**Comment by stulees** [comment]
@olegrise I've validated via manual testing of staged Image widget and checked logs. Screenshot added to the description of this PR

**Pull request merged**